croncert-config.yml

```diff
@@ -1738,6 +1738,112 @@ scrapers:
         date_location: "America/Los_Angeles"
         date_language: "en_US"
 
+  ##########
+  # Vevey
+  ##########
+  - name: CafeLitteraire
```

Comment:
I removed spaces and special characters from the names of both locations you added. The name field is more like an id that one can use to scrape a single location from a file containing a lot of different locations. After having added a location I usually check with `goskyr -c croncert-config.yml -s CafeLitteraire-stdout` So since the name can be used as value for the `-s` flag it is better if it doesn't contain any spaces or special chars.

Reply:
Thanks!